src/css/style.scss

```diff
@@ -0,0 +1,44 @@
+$blue: #0078e8;
```

Comment:
CSS reduction is considerable as we're using native Bootstrap classes and not having to style all of them ourselves.